crescent/ext/tasks/loop.py

```diff
@@ -0,0 +1,37 @@
+from typing import Callable, Sequence
+
+from crescent.ext.tasks.task import TaskCallbackT, _link_task, _Task
+from crescent.internal import MetaStruct
+
+__all__: Sequence[str] = ("loop",)
+
+
+class _Loop(_Task):
+    def __init__(self, callback: TaskCallbackT, *, hours: int, minutes: int, seconds: int) -> None:
+        self.timedelta = hours * 3600 + minutes * 60 + seconds
```

Comment:
why not use a timedelta object for this?

Reply:
i didn't want to google how it works...
probably should fix that

Reply:
it's literally what you did here
```py
x=timedelta(hours=hours, minutes=minutes, seconds=seconds)

seconds = x.total_seconds()
```